Change the paths to be /usr/bin and /usr/share instead of
/usr/local/bin and /opt

--- a/droidcam.desktop
+++ b/droidcam.desktop
@@ -2,9 +2,9 @@
 Encoding=UTF-8
 Name=DroidCam
 Comment=Use your phone as a webcam
-TryExec=/usr/local/bin/droidcam
-Exec=/usr/local/bin/droidcam
-Icon=/opt/droidcam-icon.png
+TryExec=/usr/bin/droidcam
+Exec=/usr/bin/droidcam
+Icon=/usr/share/droidcam/icon.png
 Terminal=false
 Type=Application
 Categories=Video;AudioVideo;
--- a/src/droidcam.c
+++ b/src/droidcam.c
@@ -43,7 +43,7 @@
 extern const char *thread_cmd_val_str;
 extern char snd_device[32];
 extern char v4l2_device[32];
-const char *APP_ICON_FILE = "/opt/droidcam-icon.png";
+const char *APP_ICON_FILE = "/usr/share/droidcam/icon.png";
 
 void * AudioThreadProc(void * args);
 void * VideoThreadProc(void * args);
